circle.yml

```diff
@@ -6,4 +6,9 @@ machine:
     OMNIAUTH_CLIENT_SECRET: 'omniauth_client_secret'
     OMNIAUTH_CLIENT_ID: 'omniauth_client_id'
     DEVISE_SECRET_KEY: 'devise_secret_key'
+    AWS_ACCESS_KEY_ID: 'aws_key'
```

Comment:
CircleCI env vars might be better set through their UI. It feels safer to add them there than through a Yaml file (that's how I did it for now).